elements/noflo-account.html

```diff
@@ -229,6 +272,31 @@
         logout: function () {
           this.fire('logout', true);
         },
+        relogin: function (event, details, sender) {
+          event.preventDefault();
+          var scope = sender.getAttribute('data-scope');
+          this.fire('relogin', scope);
+        },
+        userChanged: function () {
+          if (!this.user) {
+            return;
+          }
+          if (!this.user.plan || this.user.plan.type === 'free') {
```

Comment:
These are scopes we offer to the user as options to allow. The "grant access to X" buttons are populated from this.